mongo/errors.go

```diff
@@ -609,6 +609,50 @@ func (bwe BulkWriteException) HasErrorCodeWithMessage(code int, message string)
 // serverError implements the ServerError interface.
 func (bwe BulkWriteException) serverError() {}
 
+// ClientBulkWriteException is the error type returned by ClientBulkWrite operations.
+type ClientBulkWriteException struct {
+	TopLevelError *WriteError
```

Comment:
Cannot use `Error` as a field name because of the conflict with the conventional method name.